### PR TITLE
CB-584 Create minimal test run that is part of deployment pipeline

### PR DESCRIPTION
**Pull request by aszegedi**
@attilapalfi92 I've introduced the mentioned workaround as a separate commit.

@lnardai For today the related Jenkins tests are failing, because of:

```./scripts/check-results.sh
test-output/junitreports/TEST-com.sequenceiq.it.cloudbreak.newway.testcase.smoke.YarnSmokeTest.xml:    <skipped/>
test-output/junitreports/TEST-com.sequenceiq.it.cloudbreak.newway.testcase.smoke.YarnSmokeTest.xml:    <skipped/>
test-output/junitreports/TEST-com.sequenceiq.it.cloudbreak.newway.testcase.smoke.YarnSmokeTest.xml:    <skipped/>
test-output/junitreports/TEST-com.sequenceiq.it.cloudbreak.newway.testcase.smoke.YarnSmokeTest.xml:    <skipped/>
[0;91m--- !!! INTEGRATION TEST SKIPPED, CHECK [1;93mtest-output[0;91m DIR FOR RESULTS !!! ---
make: *** [check-results] Error 1
Build step 'Execute shell' marked build as failure
```

So I've removed ignored test cases from the suite.

### CODE REVIEW

**Comment by attilapalfi92** [comment]
please merge when the pipeline finished

File: docker-autoscale/Dockerfile [hunk 1]
```diff
@@ -1,4 +1,4 @@
-FROM hortonworks/hwx_openjdk:11.0-jdk-slim
```
Comment (doktoric) [inline docker-autoscale/Dockerfile]: this is not really required

File: docker-cloudbreak/Dockerfile [hunk 2]
```diff
@@ -1,4 +1,4 @@
-FROM hortonworks/hwx_openjdk:11.0-jdk-slim
```
Comment (doktoric) [inline docker-cloudbreak/Dockerfile]: this is not really required

File: docker-datalake/Dockerfile [hunk 3]
```diff
@@ -1,4 +1,4 @@
-FROM hortonworks/hwx_openjdk:11.0-jdk-slim
```
Comment (doktoric) [inline docker-datalake/Dockerfile]: this is not really required

**Pull request merged**